<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QDialog" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Eureka®</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../static/resource.qrc">
    <normaloff>:/image/image/LOGO.jpeg</normaloff>:/image/image/LOGO.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    background-color: #ffffff;
	font-style: &quot;Segoe UI&quot;;
 	color: #0B090A;
}

QLabel {
    color: #0B090A;
    font-size: 12px;
    font-weight: bold;
}

QLineEdit {
    background-color: #DFE0E2;
    padding-left: 15px;
    border-radius: 18px;
	font-size: 18px;
}

QPushButton {
    background-color: #fbba72;
    border: 2px;
    border-radius: 18px;
    font-size: 14px;
    font-weight: regular;
}

QPushButton:hover {
    background-color: #399918;
    color: #EEEEEE
}

QPushButton:pressed {
    background-color: #6703c5;
    color: #EEEEEE;
}
</string>
  </property>
  <widget class="QWidget" name="window_title_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(186, 24, 27, 255), stop:1 rgba(102, 7, 8, 255));
}</string>
   </property>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: transparent;
    color: #EEEEEE;
    font-size: 22px;
    font-weight: regular;
	font-style: &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>Pesquisar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="type_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: transparent;
    color: #EEEEEE;
    font-size: 22px;
    font-weight: regular;
	font-style: &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>{area}</string>
    </property>
   </widget>
   <widget class="QLabel" name="enaplic_logo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>4</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../static/resource.qrc">:/image/image/LOGO.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="main_area" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>641</width>
     <height>551</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
    background-color: #ffffff;
	font-style: 'Segoe UI';
}

QLabel {
    color: #0B090A;
    font-size: 11px;
    font-weight: bold;
}

QLineEdit {
    background-color: #DFE0E2;
    padding-left: 15px;
    border-radius: 18px;
	font-size: 14px;
}

QComboBox {
    background-color: #DFE0E2;
    padding: 5px 10px;
    border-radius: 18px;
    height: 20px;
    font-size: 16px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(../static/icon/arrow.png);
    width: 10px;
    height: 10px;
}

QPushButton {
    background-color: #fbba72;
    border: 2px;
    border-radius: 18px;
    font-size: 14px;
    font-weight: regular;
}

QPushButton:hover {
    background-color: #e5383b;
    color: #EEEEEE
}

QPushButton:pressed {
    background-color: #6703c5;
    color: #EEEEEE;
}</string>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
    background-color: #d60000;
    color: #EEEEEE
}</string>
    </property>
    <property name="text">
     <string> Cancelar</string>
    </property>
    <property name="icon">
     <iconset resource="../static/resource.qrc">
      <normaloff>:/icon/icon/excluir (1).png</normaloff>:/icon/icon/excluir (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="search_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_field">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ok">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
    background-color: #399918;
    color: #EEEEEE
}</string>
    </property>
    <property name="text">
     <string> OK</string>
    </property>
    <property name="icon">
     <iconset resource="../static/resource.qrc">
      <normaloff>:/icon/icon/carraca.png</normaloff>:/icon/icon/carraca.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="type_search_combobox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="currentText">
     <string>Código</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <property name="maxCount">
     <number>10</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Código</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descrição</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="type_search_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Critério</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #fff;
    border: 2px;
    border-radius: 18px;
    font-size: 14px;
    font-weight: regular;
}

QPushButton:hover {
    background-color: transparent;
    color: #EEEEEE
}

QPushButton:pressed {
    background-color: transparent;
    color: #EEEEEE;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../static/resource.qrc">
      <normaloff>:/icon/icon/lupa (2).png</normaloff>:/icon/icon/lupa (2).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="search_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
   </widget>
   <zorder>search_table</zorder>
   <zorder>btn_close</zorder>
   <zorder>search_label</zorder>
   <zorder>search_field</zorder>
   <zorder>btn_ok</zorder>
   <zorder>type_search_combobox</zorder>
   <zorder>type_search_label</zorder>
   <zorder>btn_search</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
